config/Xsession/91oidc-agent

```diff
@@ -2,11 +2,20 @@
 
 OIDCAGENT=/usr/bin/oidc-agent
 OIDCAGENTARGS=
+OIDC_AGENT_XSESSION_CONFIG_FILE="${HOME}/.config/oidc-agent-Xsession.options"
```

Comment:
This path should be `${HOME}/.config/oidc-agent/oidc-agent-Xsession.options`
Also the location of the oidc-agent directory is not fixed. You should check the following locations in that order:
- if the `$OIDC_CONFIG_DIR` env var is set use this location
- if `~/.config/oidc-agent` exists use this
- else use `~/.oidc-agent`